app/views/trips/_form.html.erb

```diff
@@ -41,6 +40,11 @@
     <%= form.number_field :pets %>
   </div>
 
+  <div>
+    <%= form.label :vehicle_id, style: "display: block" %>
+    <%= form.select(:vehicle_id, options_for_select(Vehicle.where(user_id: current_user.id).map { |elm| [elm.nickname, elm.id] })) %>
```

Comment:
interesting query, IMHO it deserves to be a `scope`, WDYT?

Reply:
yup, makes sense